### PR TITLE
Cache page size in local storage

### PR DESCRIPTION
**Pull request by sri2k1us**
- SSIA
Note: Removed 25, 50 from size options.

### CODE REVIEW

File: public/static/locales/en/util.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {
     "details": "Details",
+    "selectPageSize": "Select page size",
```
Comment (ash3rz) [inline public/static/locales/en/util.json:3]: With the addition of this, does this mean the data/apps/analyses pages also have to have `util` added to their `namespacesRequired` array? (Which reminds me, I probably need to do the same for sharing)

**Review by ash3rz — APPROVED** [review]
LGTM, just had a question about i18n stuff :shipit:

File: public/static/locales/en/util.json (hunk 1 above)
Comment (sri2k1us) [reply public/static/locales/en/util.json:3]: I don't see specific warning about including `util` when I checked for analyses now but I do see a warning about `app.js` not having `namespacesRequired` array declared. So Let me add that.

**Comment by sri2k1us** [comment]
Thanks for the review.

**Pull request merged**